resources/windows/wix/project.wxs.erb

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="windows-1252"?>
+  <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+    <Product
+      Id="*"
+      UpgradeCode="<%= settings[:upgrade_code] %>"
+      Name="<%= settings[:product_name] %>"
+      Language="1033"
+      Codepage="1252"
+      Version="<%= @version.sub(/\.g[0-9a-z]{7}$/, '') %>"
+      Manufacturer="<%= settings[:company_name] %>"
+    />
```

Comment:
You have closed the ```<Product>``` element here, but you also have a ```</Product>``` at the end of the file.
Gives this error when I try to compile it.
```
puppet-agent.wxs
C:\cygwin64\var\tmp\tmp.avLTLHrNSr\wix\puppet-agent.wxs(61) : error CNDL0104 : Not a valid source file; detail: The 'Wix' start tag on line 2 position 4 does not match the end tag of 'Product'. Line 61, position 5.
Makefile:10: recipe for target 'puppet-agent-1.3.2.389.gaa23776.1-x64.msi' failed
```